src/commands/uwu.ts

```diff
@@ -0,0 +1,46 @@
+import { FormattingPatterns } from 'discord-api-types'
+import { CommandInteraction, TextBasedChannel, TextChannel } from 'discord.js'
+import { Discord, SimpleCommand, SimpleCommandMessage, SlashOption, Slash, SimpleCommandOption } from 'discordx'
+
+
+@Discord()
+class UwU {
+
+  private general_channel = '103678524375699456';
+  private channel: TextChannel | TextBasedChannel | null = null;
+
+  private uwuify(text: string): string {
+    
+    var pattern_index:number
+
+    /* Each pattern is a tuple containing a search pattern and its associated replacement string */
+    var patterns:[RegExp,string][] = [
+      [ RegExp("r|l", "g"),              "w"     ],
+      [ RegExp("R|L", "g"),              "W"     ],
+      [ RegExp("n([aeiouAEIOU])", "g"),  "ny$1"  ],
+      [ RegExp("N([aeiou])/", "g"),      "Ny$1"  ],
+      [ RegExp("N([AEIOU])/", "g"),      "NY$1"  ]
+    ]
+
+    /* Iterate over each pattern and replace it in the user input string */
+    for (pattern_index = 0; pattern_index < patterns.length; pattern_index++)
+    {
+      text = text.replaceAll(patterns[pattern_index][0], patterns[pattern_index][1])
+    }
+
+    return text
+  }
+
+  @SimpleCommand('uwu', { description: 'UwUify text', argSplitter: '\n' })
+  simple(@SimpleCommandOption('text', { type: 'STRING' }) text: string | undefined, command: SimpleCommandMessage) {
+    if (!text) {
+      return command.sendUsageSyntax()
+    }
+
+    if (command.message.channel.id !== this.general_channel) {
```

Comment:
This command should work outside of general. I think the original limitations were if it was IN general then it had to be less than 200 characters otherwise it worked fine.